src/CssInlinerPlugin.php

```diff
@@ -74,29 +74,36 @@ public function loadOptions($options)
 
     /**
      * Find CSS stylesheet links and load them
-     * 
-     * Loads the body of the message and passes 
+     *
+     * Loads the body of the message and passes
      * any link stylesheets to $this->css
      * Removes any link elements
-     * 
+     *
      * @return string $message The message
      */
     public function loadCssFilesFromLinks($message)
     {
         $dom = new \DOMDocument();
         // set error level
         $internalErrors = libxml_use_internal_errors(true);
-        
+
         $dom->loadHTML($message);
-        
+
         // Restore error level
         libxml_use_internal_errors($internalErrors);
         $link_tags = $dom->getElementsByTagName('link');
 
         if ($link_tags->length > 0) {
             do {
                 if ($link_tags->item(0)->getAttribute('rel') == "stylesheet") {
-                    $options['css-files'][] = $link_tags->item(0)->getAttribute('href');
+                    $css_file = $link_tags->item(0)->getAttribute('href');
+                    $public_path = preg_replace('/\//', '\/', public_path());
```

Comment:
Instead of checking if the file is in the public path, I'd rather check if the file exists at its current given path.

Then if it doesn't, use the public_path, check again, and if like that it doesn't, throw an InvalidException clearly stating the file is not where it should be or the path is incorrect.